packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/VersionInfo.kt

```diff
@@ -22,7 +22,7 @@ import io.realm.kotlin.VersionId
  * Version meta data for an overall [Realm]-instance with [VersionData] for the user-facing [Realm]
  * and the underlying [SuspendableNotifier]'s and [SuspendableWriter]'s live realms.
  */
-public data class VersionInfo(val main: VersionData?, val notifier: VersionData?, val writer: VersionData?) {
-    val all: Set<VersionId> = setOf(main, notifier, writer).mapNotNull { it?.versions }.flatten().toSet()
-    val allTracked: Set<VersionId> = setOf(main, notifier, writer).mapNotNull { it?.active }.flatten().toSet()
+public data class VersionInfo(val notifier: VersionData?, val writer: VersionData?) {
```

Comment:
Why don't we track the _initial version_ anymore? Isn't it as important to know if the initial version is released as it is with the versions tracked by the notifier and writer 🤔

Reply:
I have moved it into a separate test case: `initialVersionDereferencedAfterFirstWrite`

Reply:
I removed because I was testing it now in a different test case: initialVersionDereferencedAfterFirstWrite. But it is true that it is better to keep it.